File: WebClientIS/WebClientIS/Models/Sintomas.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Net.Http;
using System.Net.Http.Headers;
using System.Text;
using System.Threading.Tasks;

namespace APIIS.Models
{
    public class Sintomas
    {
        public int IDS { get; set; }

        public string Sintoma { get; set; }

        public Sintomas()
        {
            IDS = 0;
            Sintoma = string.Empty;
        }


        public List<Sintomas> GetSintomas()
        {
            List<Sintomas> lista = new List<Sintomas>();

            HttpClient client = new HttpClient();

            client.BaseAddress = new Uri("http://6f85a52b4f05.ngrok.io/api/Sintomas");

            client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));

            HttpResponseMessage response = client.GetAsync("").Result;

            if (response.IsSuccessStatusCode)
            {
                var products = response.Content.ReadAsStringAsync().Result;

                var array = JArray.Parse(products);

                foreach (var obj in array)
                {
                    try
                    {
                        lista.Add(obj.ToObject<Sintomas>());
                    }
                    catch (Exception ex)
                    {
                        return new List<Sintomas>();
                    }
                }

            }

            return lista;
        }

        public bool CreateSintomas(string strc, Sintomas sint)
        {
            HttpClient client = new HttpClient();

            var values = new Dictionary<string, string>
            {
                { "IDS", sint.IDS.ToString() },
                {"Sintoma",sint.Sintoma }
                

            };

            client.BaseAddress = new Uri("http://6f85a52b4f05.ngrok.io/api/Sintomas");

            var jsonString = JsonConvert.SerializeObject(values);
            var content = new StringContent(jsonString, Encoding.UTF8, "application/json");

            var resposta = client.PostAsync(client.BaseAddress.ToString(), content).Result;

            if (resposta.IsSuccessStatusCode)
            {
                return true;
            }
            else
            {
                return false;
            }

        }
    }
}

File: MobileClient/sampleproject/lib/Models/Sintomas.dart
import 'dart:typed_data';
import 'package:flutter/material.dart';
import 'package:http/http.dart' as http;
import 'dart:convert' show json;
import 'package:scoped_model/scoped_model.dart';


class Sintomas extends Model{

  int ids;
  String sintoma;

  Sintomas({int idss,String sint})
  {
    this.ids = idss;
    this.sintoma = sint;
  }

  factory Sintomas.fromJson(Map<String,dynamic> json){

    return Sintomas(
      idss: json['iDS'] as int,
      sint: json['sintoma'] as String,
      
    );

  }

   
   
  Future<List<Sintomas>> getsintomas() async {
  
  http.Response response = await http.get(
    Uri.encodeFull("http://6f85a52b4f05.ngrok.io/api/Sintomas"),
    headers:{
      "Accept":"application/json"
    }
  );

  List lista = json.decode(response.body);

  List<Sintomas> listau = new List<Sintomas>();

  
  for(int i = 0; i < lista.length; i++)
    listau.add(Sintomas.fromJson(lista[i]));
  

  return listau;
}

Future<int> createSintoma(Sintomas sint) async {

  var url ='http://6f85a52b4f05.ngrok.io/api/Sintomas';
  var body = json.encode(<String, String>{
      'IDS': sint.ids.toString(),
      'Sintoma':sint.sintoma    

    });

  print(body);



  http.Response response = await http.post(url,
      headers: {"Content-Type": "application/json"},
      body: body
  );

  return response.statusCode;

}

}